docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Building-Quickstart.md

```diff
@@ -29,137 +28,136 @@ Will give you a list of all commands available to you. Use
 
 to see the in-game help for that command.
 
-## Looking around
+## Looking Around
 
-The most common comman is
+The most common command is
 
     look
 
-This will show you the description of the current location. `l` is an alias.
+This will show you the description of the current location. `l` is an alias for the look command.
 
-When targeting objects in commands, you have two special labels you can use, `here` for the current room or `me`/`self` to point back to yourself. So
+When targeting objects in commands, you have two special labels you can use: (1) `here` for the current room, or (2) `me`/`self` to point back to yourself. Thus,
```

Comment:
From experience, the (1) and (2) just confuses newcomers to MU* syntax - they think stuff like this is part of the command. I may be better to make a numbered list or bullet-point list here.

Reply:
ah! yes, that could certainly be mistaken for code. ty.